app/config.go

```diff
@@ -273,7 +285,8 @@ func InitTendermint() *node.Node {
 	})
 	logger, err := flags.ParseLogLevel(GlobalConfig.TendermintConfig.LogLevel, logger, "info")
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
```suggestion
              log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work